apps/nfid-frontend/src/features/activity/index.tsx

```diff
@@ -12,6 +12,9 @@ const ActivityPage = () => {
   const { data: activeTokens = [], isLoading: isActiveLoading } = useSWR(
     "activeTokens",
     fetchActiveTokens,
+    {
+      revalidateOnFocus: false,
+    },
   )
 
   return (
```

Comment:
is it possible to init tokens here? also to calculate total usd balance here and just pass props down to components? will be much easier to understand whats going on here, because logic will be in one place, it's definitely not responsibility of tokensUsdValue to init tokens